js/web/lib/wasm/wasm-common.ts

```diff
@@ -28,6 +28,16 @@ export const enum DataType {
   bfloat16 = 16
 }
 
+/**
+ * DataType in WGSL.
+ */
+export enum WGSLDataType {
+  undefined = 0,
+  f32 = 1,
+  i32 = 6,
+  u32 = 12,
+}
```

Comment:
how about using a function like `getWgslType(tensorType: number)` so that we can check if a param is invalid. And we can use const enum so that typescript will not emit extra code

Reply:
Done.